metagraph/src/common/utils/simd_utils.hpp

```diff
@@ -5,6 +5,10 @@
 #include <immintrin.h>
 #endif
 
+#ifdef __SSE4_1__
+#include <smmintrin.h>
+#endif
+
 #ifdef __SSE2__
```

Comment:
should it be `#elif`?

Reply:
Not sure if it matters too much since these headers have include guards.

Reply:
apparently including `x86intrin.h` is enough for all compilers, except MSVC, so I just used that